ci-operator/step-registry/ipi/conf/gcp/ipi-conf-gcp-commands.sh

```diff
@@ -87,6 +87,9 @@ compute:
       type: ${COMPUTE_NODE_TYPE}
 EOF
 
+# Add "onHostMaintenance" to "Terminate" to all the nodes having the gcp type as gpu
```

Comment:
Can we opt-in `ipi-conf-gcp-confidential-computing` by setting proper env var in `e2e-gpu` ci job to save the change here ?

Reply:
We can file a quick fix PR to enhance it to meet your requirements. The reason why I do not suggest to update `onHostMaintenance` per instance type in `ipi-config-gcp`, because sometimes users may not set it on purpose even if the instance type has `gpu` keyword.

Reply:
https://github.com/openshift/release/pull/67948